src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustDetailsValidatorTest.java

```diff
@@ -292,11 +310,76 @@ void testErrorReportedWhenUnableToObtainAllTrustInfoFieldIsNull() {
+        String validationMessage = ValidationMessages.NOT_NULL_ERROR_MESSAGE.replace("%s", qualifiedFieldName);
+        assertError(qualifiedFieldName, validationMessage, errors);
+    }
+
+    @Test
+    void testNoErrorReportedWhenTrustNotInvolvedInOverseasEntityAndCeasedDateSameAsCreationDate() {
+        trustDataDtoList.get(0).setTrustInvolvedInOverseasEntity(false);
+        trustDataDtoList.get(0).setCreationDate(LocalDate.of(2020,4, 11));
+        trustDataDtoList.get(0).setCeasedDate(LocalDate.of(2020,4, 11));
+        Errors errors = trustDetailsValidator.validate(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT, true);
+        assertFalse(errors.hasErrors());
+    }
+
+    @Test
+    void testNoErrorReportedWhenTrustNotInvolvedInOverseasEntityAndCeasedDateAfterCreationDate() {
+        trustDataDtoList.get(0).setTrustInvolvedInOverseasEntity(false);
+        trustDataDtoList.get(0).setCreationDate(LocalDate.of(2020,4, 10));
+        trustDataDtoList.get(0).setCeasedDate(LocalDate.of(2020,4, 11));
+        Errors errors = trustDetailsValidator.validate(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT, true);
+        assertFalse(errors.hasErrors());
+    }
+
+    @Test
+    void testErrorReportedWhenTrustNotInvolvedInOverseasEntityAndCeasedDateBeforeCreationDate() {
+        trustDataDtoList.get(0).setTrustInvolvedInOverseasEntity(false);
+        trustDataDtoList.get(0).setCreationDate(LocalDate.of(2020,4, 11));
+        trustDataDtoList.get(0).setCeasedDate(LocalDate.of(2020,4, 10));
+        Errors errors = trustDetailsValidator.validate(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT, true);
+        String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.TRUST_DATA, TrustDataDto.CEASED_DATE_FIELD);
+        String validationMessage = ValidationMessages.CEASED_DATE_BEFORE_CREATION_DATE_ERROR_MESSAGE.replace("%s", qualifiedFieldName);
+        assertError(qualifiedFieldName, validationMessage, errors);
+    }
+
+    @Test
+    void testErrorReportedWhenTrustNotInvolvedInOverseasEntityAndCeasedDateIsInTheFuture() {
+        trustDataDtoList.get(0).setTrustInvolvedInOverseasEntity(false);
+        trustDataDtoList.get(0).setCreationDate(LocalDate.of(2020,4, 11));
+        trustDataDtoList.get(0).setCeasedDate(LocalDate.now().plusDays(1));
+        Errors errors = trustDetailsValidator.validate(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT, true);
+        String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.TRUST_DATA, TrustDataDto.CEASED_DATE_FIELD);
+        String validationMessage = ValidationMessages.DATE_NOT_IN_PAST_ERROR_MESSAGE.replace("%s", qualifiedFieldName);
+        assertError(qualifiedFieldName, validationMessage, errors);
+    }
+
+    @Test
+    void testErrorReportedWhenTrustNotInvolvedInOverseasEntityAndCeasedDateBeforeAnyIndividualsDateOfBirth() {
```

Comment:
I'm wondering how this test differs from the other new one you added above - `testErrorReportedWhenCeasedDateIsBeforeIndividualsDateOfBirth`? I've not compare the lines of code but the names do sound similar.

Reply:
First test: 
testErrorReportedWhenCeasedDateIsBeforeIndividualsDateOfBirth

Second test: testErrorReportedWhenTrustNotInvolvedInOverseasEntityAndCeasedDateBeforeAnyIndividualsDateOfBirth 
with:   trustDataDtoList.get(0).setTrustInvolvedInOverseasEntity(false);

Reply:
Ah right, I think I understand. So by default, in the first test, the trust is still involved in the OE and in the second it's not.

But... if the trust _is_ still involved (so the first test scenario), should the API validation even be checking the ceased dates of individuals and raising a `CEASED_DATE_BEFORE_INDIVIDUALS_DATE_OF_BRITH_ERROR_MESSAGE` error (as it's doing on line 243)?

Reply:
Updated the tests some have been removed and replaced further down in the suite as they are no longer applicable when the new flag checking trust involvement in the OE as a whole is true. All these checks will now need to be when the new flag is false.